ebuild-writing/variables/text.xml

```diff
@@ -588,5 +588,54 @@ when joining paths. For example: <c>${D%/}/</c>, <c>${ED%/}/</c>,
 </body>
 </section>
 
+<section>
+<title>Use of constant-value variables in ebuilds</title>
+<body>
+
+<p>
+Variables frequently have significant value in ebuilds, making it
+possible to avoid unnecessary repetitions and make maintenance easier.
+However, references to constant-value variables should be used with care
+as their proactive use can harm readability and increase maintenance
+burden (e.g. when renaming the package). In particular, using variables
+whose values are not strictly guaranteed to match the expected string
+should be avoided.
```

Comment:
The idea is that it's fine to use a variable if it's 100% the same thing — i.e. you use PV to indicate upstream version, so it's fine to use PV when you mean upstream version. However, PN is Gentoo package name, so it's not fine to randomly replace upstream paths which are not in ebuild control with PN. Can you think of a better way of saying that?